web/src/js/urlState.ts

```diff
@@ -70,9 +70,9 @@ export function updateUrlFromStore(store) {
 
     let url;
     if (state.flows.selected.length > 0) {
-        url = `/flows/${state.flows.selected[0]}/${state.ui.flow.tab}`;
+        url = `flows/${state.flows.selected[0]}/${state.ui.flow.tab}`;
```

Comment:
This breaks all existing links to mitmweb instances, which I'd like to avoid. location.hash does not need relative URLs.